Pocket/pocket-ios
# SwiftUI Hero View and general SwiftUI Home structure

## Goal
This PR adds the following `SwiftUI` Home elements:
- Hero View
- Save Button
- WIP Slate View, which represents a component in the Home View
- WIP Home View, that leverages the new view(s)

Additionally
- Made changes to `SwifData` classes to expose their propertys to `Sync` consumers
- Adds a temporary fourth tab to see the progress of the `SwiftUI` Home
- Temporarily sets the default tab to the new `SwiftUI` Home


## Test Steps
* If you're curious, build and run, and you'll see the new home, still rough and wip.

## Screenshots
- Not yet the time for it
